pulumi/src/statefulService.ts

```diff
@@ -59,6 +59,18 @@ export function createCoinService(args: CoinServiceArgs, assetName: string, snap
         ...(args.memoryRequest && { memory: args.memoryRequest }),
       },
     },
+    ...(args.readinessEndpoint && {
```

Comment:
I think we should leverage the existing `readinessProbe` config and set on the container  if `(!readiness && { ...args.readinessProbe })`, we can update `init`, `liveness`, and `readiness` to `initFile`, `livenessFile` and `readinessFile` to clarify as well